### PR TITLE
Lvpyt 14 player fixed

### PR DESCRIPTION
**Pull request by TarasStankovskyi**

### CODE REVIEW

File: itaplay/itaplay/player/views.py [hunk 1]
```diff
@@ -10,17 +10,18 @@
 from django.views.generic import DeleteView
 from django.views.decorators.csrf import csrf_protect
 from django.shortcuts import render, render_to_response
-from django.http import HttpResponseRedirect, HttpResponse
+from django.http import HttpResponseRedirect, HttpResponse,HttpResponseBadRequest
```
Comment (lhalam) [inline itaplay/itaplay/player/views.py]: add space

File: itaplay/itaplay/monitor/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (lhalam) [inline itaplay/itaplay/monitor/tests.py]: delete this file

File: itaplay/itaplay/monitor/urls.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import patterns, url
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.GetMonitorView.as_view(), name='monitor'),
+    url(r'^get_by_mac/(?P<mac>(\w\:?)+)/$', views.MonitorView.as_view()),
+]
```
Comment (lhalam) [inline itaplay/itaplay/monitor/urls.py]: add end line

File: itaplay/itaplay/projects/views.py [hunk 4]
```diff
@@ -32,7 +32,8 @@ def post(self, request):
             for clip in area['clips']:
                 clip_tag = ElementTree.SubElement(template_area, 'clip')
                 clip_tag.set('id',str(clip['pk']))
```
Comment (lhalam) [inline itaplay/itaplay/projects/views.py:34]: add a space

File: itaplay/itaplay/static/js/app/authentification/controllers/header-controller.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+angular.module('LogoutController',['ngMaterial'])
+.controller('LogoutController', function($scope,$http, $window) {
+
+    $scope.LogoutUser = function() {
+        $http.get('/auth/logout').then(function(response) {
+            $window.location.href = '/';
```
Comment (lhalam) [inline itaplay/itaplay/static/js/app/authentification/controllers/header-controller.js]: add tabs

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            if (window.DOMParser) {
+                parser = new DOMParser();
+                xmlDoc = parser.parseFromString(response.data.template, "text/xml");
+                }
+            else { // Internet Explorer
+
+                xmlDoc = new ActiveXObject("Microsoft.XMLDOM");
+                xmlDoc.async = false;
+                xmlDoc.loadXML(response.data.template);
+                }
```
Comment (lhalam) [inline itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: delet one tab

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+                return $sce.trustAsResourceUrl(src);
+            }
+
+    $scope.currentIndex=[];
+    $scope.currentVideoIndex = [];
+
+
+    $scope.isCurrentSlideIndex = function (id_area, index) {
+        return $scope.currentIndex[id_area] === index;
+
```
Comment (lhalam) [inline itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: delete empty line

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    };
+
+    $scope.isCurrentSlideVideoIndex = function(id_area, index) {
+        return $scope.currentVideoIndex[id_area] === index;
+
```
Comment (lhalam) [inline itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: delete empty line

File: itaplay/itaplay/static/js/app/projects/controllers/projects-controller.js [hunk 9]
```diff
@@ -16,7 +16,7 @@ function AddProjectTemplateController ($scope,$routeParams, $http, $location, $m
             $scope.data = "Something went wrong";
         });
 
-        $http.get("/clips/clips/").then(function (response) {
+        $http.get("/clips/allclips/").then(function (response) {
             $scope.clips = response.data;
         }, function (response) {
             console.log(response);
```
Comment (lhalam) [inline itaplay/itaplay/static/js/app/projects/controllers/projects-controller.js]: delete empty console.log

File: itaplay/itaplay/static/styles/login.css [hunk 10]
```diff
@@ -49,3 +49,14 @@ form {
...
+    border-radius: 1%;
+    margin = 50px;
+    background-color: rgba(164, 198, 193, 0.8);
+}
+
+.content {
+    background-color: #FFFCF4;
+    border-radius: 1%;
+    margin = 50px;
+}
```
Comment (lhalam) [inline itaplay/itaplay/static/styles/login.css]: add end line

File: itaplay/itaplay/static/styles/logout.css [hunk 11]
```diff
@@ -0,0 +1,9 @@
+.md-button.logo {
+   height: 7rem;
+   width: 7rem;
+ }
+  .png-icon{
+   height: 100%;
+   width: 100%;
+   border-radius: 30px;
+}
```
Comment (lhalam) [inline itaplay/itaplay/static/styles/logout.css]: add end line

File: itaplay/itaplay/test.py [hunk 12]
```diff
@@ -0,0 +1,4 @@
+# test.py
```
Comment (lhalam) [inline itaplay/itaplay/test.py]: delete this file

File: itaplay/itaplay/tests/player/test_views.py [hunk 13]
```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
```
Comment (lhalam) [inline itaplay/itaplay/tests/player/test_views.py:3]: correct group imports

File: itaplay/itaplay/player/views.py (hunk 1 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/player/views.py]: done

File: itaplay/itaplay/monitor/tests.py (hunk 2 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/monitor/tests.py]: done

File: itaplay/itaplay/monitor/urls.py (hunk 3 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/monitor/urls.py]: done

File: itaplay/itaplay/static/js/app/authentification/controllers/header-controller.js (hunk 5 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/js/app/authentification/controllers/header-controller.js]: done

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js (hunk 6 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: done

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js (hunk 7 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: done

File: itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js (hunk 8 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/js/app/monitor/controllers/monitor-controller.js]: done

File: itaplay/itaplay/static/js/app/projects/controllers/projects-controller.js (hunk 9 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/js/app/projects/controllers/projects-controller.js]: done

File: itaplay/itaplay/static/styles/login.css (hunk 10 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/styles/login.css]: done

File: itaplay/itaplay/static/styles/logout.css (hunk 11 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/static/styles/logout.css]: done

File: itaplay/itaplay/test.py (hunk 12 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/test.py]: done

File: itaplay/itaplay/tests/player/test_views.py (hunk 13 above)
Comment (TarasStankovskyi) [reply itaplay/itaplay/tests/player/test_views.py:3]: done

**Pull request merged**